move AliasWithNameOp to caffe2/operators

wat3rBro opened the pull request:
Differential Revision: D18871766

kostmo commented:
## CircleCI build failures summary
As of commit a4ef85b:
* **1/1** broken upstream at merge base 679b20b (see [grid view](https://dr.pytorch.org/master-timeline.html?count=1&sha1=679b20b1e4b79dd3d62d8a8cb625efeb5a51fe29&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35))
    * You may want to rebase on the `viable/strict` branch (see its [recency history](https://dr.pytorch.org/master-viable-commits.html)):
        * If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
          ```
          git fetch viable/strict
          git rebase --onto viable/strict $(git merge-base origin/master HEAD)
          ```
        * If your commit is older than `viable/strict`:
          ```
          git fetch viable/strict
          git rebase viable/strict
          ```
* **0/1** failures introduced in this PR



## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=a4ef85bb15294368b5bb43366d794c682a78b643).

### 1 failure *not* recognized by patterns:
| Job | Step |
| --- | --- |
| ![CircleCI](https://avatars0.githubusercontent.com/ml/7?s=12) <sup>pytorch_windows_build</sup> | <sup>Build</sup> |


---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

wat3rBro committed:
move AliasWithNameOp to caffe2/operators

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/31010

Differential Revision: D18871766

fbshipit-source-id: 469da829496e2a3c430e9a08ede801dad5caa68e
